stdlib/Logging/src/ConsoleLogger.jl

```diff
@@ -58,10 +58,11 @@ function default_logcolor(level)
 end
 
 function default_metafmt(level, _module, group, id, file, line)
-    color = default_logcolor(level)
+    prefix_color = default_logcolor(level)
     prefix = (level == Warn ? "Warning" : string(level))*':'
     suffix = ""
-    Info <= level < Warn && return color, prefix, suffix
+    suffix_color = Base.stackframe_lineinfo_color()
```

Comment:
The idea with `light_black` was to not distract people too much with the logging metadata and I'd rather like to preserve that. But `stackframe_lineinfo_color` is `:bold` which is the opposite.

I'm also not sure why we'd choose `stackframe_lineinfo_color` here, as it's not clearly related to logging. We could just make a new color category and matching environment variable and set it default to `:light_black`.

Reply:
I originally set the default `suffix_color` to `:light_black`, which I thought was preferable. But other people wanted to emphasize the similarities between the *mini stacktrace* and a full stacktrace.